go.mod

```diff
@@ -1,47 +1,48 @@
 module github.com/siderolabs/image-factory
 
-go 1.22.6
+go 1.23.3
 
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/google/go-containerregistry v0.20.2
 	github.com/h2non/filetype v1.1.3
 	github.com/julienschmidt/httprouter v1.3.0
-	github.com/klauspost/compress v1.17.9
+	github.com/klauspost/compress v1.17.11
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/prometheus/client_golang v1.20.2
-	github.com/siderolabs/gen v0.5.0
+	github.com/prometheus/client_golang v1.20.5
+	github.com/siderolabs/gen v0.7.0
 	github.com/siderolabs/go-debug v0.4.0
 	github.com/siderolabs/go-pointer v1.0.0
-	github.com/siderolabs/talos v1.8.0-alpha.2
-	github.com/siderolabs/talos/pkg/machinery v1.8.0-alpha.2.0.20240830181738-6f7c3a8e5c63
+	github.com/siderolabs/talos v1.9.0-alpha.2
+	github.com/siderolabs/talos/pkg/machinery v1.9.0-alpha.2.0.20241113103058-8b4253d18544
```

Comment:
we need to flush caches for secureboot ISOs I guess (if you remember we had a cache hash calculation function we could use to inject errata)